apps/frontend/src/components/ProjectListItem.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { ProjectInterface } from '@datatlas/shared/models';
+import { NormalizedProjectInterface } from '@datatlas/shared/models';
 
-export type ProjectListItemProps = ProjectInterface;
+export type ProjectListItemProps = NormalizedProjectInterface;
 
-export const ProjectListItem = ({ id, name }: ProjectListItemProps) => {
+export const ProjectListItem = ({ id, name, draft }: ProjectListItemProps) => {
   return (
     <article key={id}>
       <h2>{name}</h2>
+      {draft && <h3>Draft</h3>}
```

Comment:
I don't think we need to add a title for the draft project, because there are display with the mention 'draft' or 'published'.

Reply:
You're probably right, you'll make this looks nice & all once this PR is merged.